LICENCE

```diff
@@ -0,0 +1,9 @@
+/*******************************************************
+ * Copyright (C) 2017-2018 James Thorogood <jamesthorogood@ewb.ca>
+ *
+ * This file is part of AgriBoard.
+ * All files in this directory and all subdirectories are also part of AgriBoard.
+ *
+ * AgriBoard and its enclosing files can not be copied and/or distributed without
```

Comment:
cannot is one word

Reply:
According to this link, both uses are acceptable.
https://en.oxforddictionaries.com/usage/cannot-or-can-not